app/views/content/non-uk-teachers/train-to-teach-in-england-as-an-international-student.md

```diff
@@ -60,40 +60,30 @@ keywords:
 ---
 
 
+Citizens of every country in the world can apply to train to teach in England.
 
-Citizens of every country in the world can apply to train to teach in England. When you plan your training, bear in mind that:
+When you have successfully completed your training, you will be awarded English [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts), a professional qualification valued globally.
 
-- international candidates can expect to pay an average of £14,765 to train, plus
-living costs
-- depending on your immigration status and the subject you train to teach, you may get some financial support
-- training lasts one academic year full-time, or longer part-time
-- you can apply for paid teaching jobs as soon as you successfully complete your training and gain English qualified teacher status (QTS)
-- you'll earn at least £30,000 a year (or more in London) when you qualify as a teacher 
-- you can update, extend or switch your visa to live and work in the UK long-term
-
-## Already a qualified teacher?
-
-If you're already a qualified teacher, find out how you can [teach in England if you completed your teacher training outside the UK](/non-uk-teachers/teach-in-england-if-you-trained-overseas).
-
-## Why train to teach in England?
+QTS will allow you to apply for teaching jobs in all schools in England. When you get a teaching job, depending on your immigration status, you may also need to [apply for a visa allowing you to work in the UK](/non-uk-teachers/visas-for-non-uk-teachers).   
 
-Teacher training in England is practical, hands-on and supportive. As an international applicant, you’ll have the opportunity to train at a world class institution where creativity and innovation is valued, gaining lots of classroom experience along the way.
+### Meet non-UK citizens who trained to teach in England
 
-When you have successfully completed your training, you will be awarded [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts), a professional qualification valued globally.
+Read a [transcript of this video](/non-uk-teachers/why-train-to-teach-in-england-video-transcript).
```

Comment:
Would suggest this is a bit more descriptive for screen reader users - previously we had:

[Read the transcript for the 'Why train to teach in England?' video]

Reply:
The transcript should also come below the video